### DISCUSSION TITLE
how to use cli?

### DISCUSSION THREAD

**Discussion by dwyane20** [pawelsalawa/sqlitestudio#4157] [Q&A]
### Details

can I use sqlitestudiocli connect WxSQLite3 and export tables to csv?

I tried, but it just can connect sqlite3 
I found cli in wiki but there is nothing

**Comment by pawelsalawa** [comment]
To be honest I don't remember if the CLI is capable of using different plugins for databases. I think it's not.
Be aware, that CLI is still in experimental stage, it was not maintained for quite some time and it's not best idea to use it yet.

**Reply by dwyane20** [reply]
Thank you, GUI is very good,I want to learn more about CLI, can you give  some suggestions？information about CLI in wiki is too little

**Reply by pawelsalawa** [reply]
When you run it, there is a `.help` command.
Regarding the startup options and arguments, you can see them by running `sqlitestudiocli.exe --help` (or whatever the executable is named on your operating system).

**Reply by pawelsalawa** [reply]
Oh, and you can get more details about particular comands, like this: `.help dblist`

**Reply by dwyane20** [reply]
thank you very much
